diff --git a/qa/config/selenium_test.conf b/qa/config/selenium_test.conf
index f0f4ea4..b6a795b 100644
--- a/qa/config/selenium_test.conf
+++ b/qa/config/selenium_test.conf
@@ -16,9 +16,9 @@
     'host'                              => 'localhost',
     'port'                              => 4444,
     'browser_url'                       => 'http://localhost',
-    'attachment_file'                   => '/home/bugzilla/devel/htdocs/bmo/qa/config/patch.diff',
+    'attachment_file'                   => '/var/www/html/bmo/qa/config/patch.diff',
     'bugzilla_installation'             => 'bmo',
-    'bugzilla_path'                     => '/home/bugzilla/devel/htdocs/bmo',
+    'bugzilla_path'                     => '/var/www/html/bmo',
     'test_bug_1'                        => 1,
     'test_bug_2'                        => 2,
     'admin_user_login'                  => 'admin@mozilla.test',
